cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/KafkaCruiseControlServlet.java

```diff
@@ -847,6 +849,12 @@ private boolean wantJSON(HttpServletRequest request) {
     return Boolean.parseBoolean(jsonString);
   }
 
+  private boolean allowCapacityEstimation(HttpServletRequest request) {
```

Comment:
What if the passed string cannot be parsed as a boolean.

Reply:
The behavior in this case is determined by the `Boolean#parseBoolean(String)` -- i.e. will be `false` for anything except `True` (not case sensitive).